vllm/worker/model_runner_base.py

```diff
@@ -265,13 +264,13 @@ def prepare_model_input(
         """
         raise NotImplementedError
 
-    @current_platform.inference_mode()
```

Comment:
This is removed directly. It's useless now?

Reply:
it is moved to https://github.com/vllm-project/vllm/pull/11602/files#diff-1ca8953d4dab62fff08a34c8cf370d2a37aeb009526ab4fc5464a65e1d03b036R84 .

this line does nothing actually. no subclass calls this function.